Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -189,6 +189,9 @@ uplink-observation-kit-desc = Includes surveillance camera monitor board and sec
 uplink-emp-kit-name = Electrical Disruptor Kit
 uplink-emp-kit-desc = The ultimate reversal on energy-based weaponry: Disables disablers, stuns stunbatons, discharges laser guns! Contains 3 EMP grenades and an EMP implanter. Note: Does not disrupt actual firearms.
 
+uplink-decoy-kit-name = Decoy Kit
+uplink-decoy-kit-desc = State of the art distraction technology straight from RND. Comes with five realistic balloons, four decoy grenades, and some snap poppers!
```

Comment:
`uplink-decoy-kit-desc = State-of-the-art distraction technology straight from RND. Comes with five realistic balloons, four decoy grenades, and some snap poppers!`